content_script.js

```diff
@@ -17,6 +17,27 @@ class HTMLSerializer {
     this.FILTERED_TAGS = new Set(['script', 'noscript', 'style', 'link']);
 
     /**
+     * @private {Set<string>} Contains the tag names for elements
+     *     that have no closing tags.
```

Comment:
Cool, can you just update the comment to say the list is from the spec? Just add a link to https://html.spec.whatwg.org/multipage/syntax.html#void-elements.

Don't worry about whatwg vs w3c specs. It's a long and terribly sad story.. When browsers follow specs at all, they follow both.

With the comment updated, this patch LGTM.